cit-api/pipeline/migrations/0143_auto_20210406_2301.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 2.2.16 on 2021-04-06 23:01
+
+import django.contrib.gis.db.models.fields
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('pipeline', '0142_auto_20210405_1733'),
```

Comment:
There was recent migration merged in, we'll need to update the reference and file name, or make a merge migration.